lib/ezapi_client/services/gen_data.rb

```diff
@@ -44,9 +44,16 @@ def default_command
 
     def default_json
       message.each_with_object({}) do |(key, value), hash|
-        hash[key.to_s.camelcase(:lower)] = value
+        hash[key.to_s.camelcase(:lower)] = strip_special_characters(value)
       end.to_json
     end
 
+    def strip_special_characters(value)
+      return value if !value.is_a?(String)
+
+      rule = /[^0-9A-Za-z ]/
```

Comment:
This looks like it should be a frozen constant :+1:

We should allow `-` though.

```ruby
/[^\w-\s]/
```

☝️  "not a digit or letter, not a hyphen, not a space"